content/source/docs/enterprise/logging/index.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "enterprise2"
+page_title: "Terraform Enterprise Logging"
+sidebar_current: "docs-enterprise2-home"
+---
+
+## Terraform Enterprise Logs
+
+This document contains information about interacting with Private Terraform Enterprise logs.
+
+### Audit Logs
+
+As of Private Terraform Enterprise release v201802-1, audit logging is available in Private Terraform Enterprise. 
+
+#### What will be logged?
```

Comment:
Since this isn't in a formal FAQ anymore, I suggest rephrasing headers as nouns or statements. e.g. "Log Contents," "Log Format," "Log Location"